docs/getting-started/deployment.md

```diff
@@ -0,0 +1,275 @@
+```
+
+#### Lock allowed images list to only allow "busybox" and "nginx"
+
+```
+LOCKED=~/Library/Preferences/io.rancherdesktop.profile.locked.plist
+plutil -create xml1 "$LOCKED"
+
+plutil -insert containerEngine -dictionary "$LOCKED"
+plutil -insert containerEngine.enabled -bool true "$LOCKED"
+
+plutil -insert containerEngine.patterns -array "$LOCKED"
+plutil -insert containerEngine.patterns -string busybox -append "$LOCKED"
+plutil -insert containerEngine.patterns -string nginx -append "$LOCKED"
+```
+
+#### Verify the plist files
+
+```xml title="~/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>containerEngine</key>
+	<dict>
+		<key>name</key>
+		<string>moby</string>
+	</dict>
+	<key>kubernetes</key>
+	<dict>
+		<key>enabled</key>
+		<false/>
+	</dict>
+</dict>
+</plist>
+```
+
+```xml title="~/Library/Preferences/io.rancherdesktop.profile.locked.plist"
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>containerEngine</key>
+	<dict>
+		<key>enabled</key>
+		<true/>
+		<key>patterns</key>
+		<array>
+			<string>busybox</string>
+			<string>nginx</string>
+		</array>
+	</dict>
+</dict>
+</plist>
+```
+
+</TabItem>
+<TabItem value="Linux">
+
+On Linux the deployment profiles are stored in JSON format.
+
+The locations for the profiles are:
+
+```
+/etc/rancher-desktop/defaults.json
+/etc/rancher-desktop/locked.json
+~/.config/rancher-desktop.defaults.json
+~/.config/rancher-desktop.locked.json
+```
+
+#### Convert all current settings into a deployment profile
+
+Since deployment profiles are stored in JSON format, the simplest way to create them is by saving the current application settings to the profile location, and then fine-tuning the profile with a text editor.
+
+```
+rdctl list-settings > ~/.config/rancher-desktop.defaults.json
+```
+
+#### By default use the "moby" container engine and disable Kubernetes
+
+```json title="~/.config/rancher-desktop.defaults.json"
+{
+  "containerEngine": {
+    "name": "moby"
+  },
+  "kubernetes": {
+    "enabled": true
+  }
+}
+```
+
+#### Lock allowed images list to only allow "busybox" and "nginx"
+
+```json title="~/.config/rancher-desktop.locked.json"
+{
+  "containerEngine": {
+    "allowedImages": {
+      "enabled": true,
+      "patterns": ["busybox","nginx"]
+    }
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Known Issues and Limitations
+
+* The `containerEngine.allowedImages` settings are currently the only ones that can be locked.
+* On macOS incorrectly formatted profiles are ignored instead of preventing the app from loading.
+* There is no way to set `diagnostics.showMuted` (and on Windows `WSL.integrations`) via deployment profile.
```

Comment:
The same holds for `extensions`, although I don't know if we plan to make that field directly writable, or only via some plugin mechanism.

Reply:
There should be no `extensions` setting in `release-1.8`. It is part of RDX and should only exist on `main`.